dropshot/data/dont_use_this_cert.pem

```diff
@@ -0,0 +1,113 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I believe that lines outside the BEGIN and END lines will be ignored in PEM files.  Can we add some comments explaining where this file came from and how it's used?

Even better, if this is used for testing, would it make sense to generate it in the test suite rather than checking it in?  That way someone wouldn't have to figure out how to regenerate it when we need to change it -- the process will be automated and run all the time.

Reply:
Removed these and switched to generating self-signed certificates in the tests